blocks/small-manual-promo-block/features/small-manual-promo/default.test.jsx

```diff
@@ -1,118 +1,120 @@
 import React from "react";
-import { mount } from "enzyme";
+import { render, screen } from "@testing-library/react";
 import SmallManualPromo from "./default";
 
 jest.mock("@wpmedia/engine-theme-sdk", () => ({
-	Image: ({ url }) => <img src={url} alt="fake test image" />,
-	LazyLoad: ({ children }) => <>{children}</>,
-	isServerSide: () => true,
+	LazyLoad: ({ children }) => children,
 }));
-jest.mock("fusion:themes", () => jest.fn(() => ({})));
-jest.mock("fusion:properties", () => jest.fn(() => ({})));
-jest.mock("fusion:properties", () => jest.fn(() => ({})));
-jest.mock("fusion:context", () => ({
-	useFusionContext: jest.fn(() => ({})),
-	useComponentContext: jest.fn(() => ({
-		registerSuccessEvent: () => ({}),
-	})),
+jest.mock("@wpmedia/arc-themes-components", () => ({
+	formatURL: jest.fn(),
+	Heading: ({ children }) => <h1>{children}</h1>,
+	HeadingSection: ({ children }) => children,
+	Image: (props) => {
+		const { url, "data-aspect-ratio": dataAspectRatio } = props;
+		return <img src={url} alt="test image" data-aspect-ratio={dataAspectRatio} />;
+	},
+	isServerSide: jest.fn(() => true),
+	Link: ({ children }) => children,
+	MediaItem: ({ children }) => children,
+	Stack: ({ children }) => <article>{children}</article>,
 }));
-jest.mock("fusion:content", () => ({
-	useContent: jest.fn(() => ({})),
-	useEditableContent: jest.fn(() => ({
-		searchableField: () => {},
+
+jest.mock("fusion:context", () => ({
+	useFusionContext: jest.fn(() => ({
+		isAdmin: false,
 	})),
 }));
 
-const config = {
-	showHeadline: true,
-	showImage: true,
+const customFields = {
 	headline: "This is the headline",
+	imagePosition: "right",
+	imageRatio: "3:2",
 	imageURL: "www.google.com/fake.png",
+	lazyLoad: false,
 	linkURL: "www.google.com",
+	newTab: true,
+	showHeadline: true,
+	showImage: true,
 };
 
-describe("the small promo feature", () => {
+describe("the small manual promo feature", () => {
 	afterEach(() => {
 		jest.resetModules();
 	});
 
 	beforeEach(() => {
 		jest.mock("fusion:context", () => ({
 			useFusionContext: jest.fn(() => ({
-				arcSite: "the-sun",
-				id: "testId",
+				isAdmin: false,
 			})),
 		}));
 	});
 
+	/*
 	it("should have 1 container fluid class", () => {
```

Comment:
If test is no longer valid we can delete the code

Reply:
I thought I got them all but must have mixed these.
Got it. Thanks!